modules/oadp-cleaning-up-after-data-mover-snapshots.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc
+
+:_content-type: PROCEDURE
+[id="oadp-cleaning-up-after-data-mover-snapshots_{context}"]
+= Deleting snapshots in a bucket
+
+Data Mover might leave one or more snapshots in a bucket after a backup. You can either delete all the snapshots or delete individual snapshots.
+
+.Procedure
+
+. To delete all snapshots in your bucket, delete the `/<protected-ns>` folder that is specified in the DPA `.spec.backupLocation.objectStorage.bucket`.
+. To delete an individual snapshot:
+.. Browse to the `/<protected-ns>` folder that is specified in the DPA `.spec.backupLocation.objectStorage.bucket`.
```

Comment:
Underscores for replaceable values; what is DPA? Can you spell that out?

Also, should add "field" or something after the `.spec.backupLocation.objectStorage.bucket` field name. (Here and above)

Reply:
@kaovilai @bergerhoffer Continuing from above... should <protected-ns> be changed to <app_namespace> ?